modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/logic/UIItemsBuilder.java

```diff
@@ -701,7 +701,7 @@ public void addMoveToTheRecycleBinToolbarItem(
 		_addJavaScriptUIItem(
 			new JavaScriptToolbarItem(), toolbarItems,
 			DLUIItemKeys.MOVE_TO_THE_RECYCLE_BIN,
-			LanguageUtil.get(_resourceBundle, "move-to-recycle-bin"),
+			LanguageUtil.get(_resourceBundle, "delete"),
```

Comment:
Here we may also want to rename the method ( _addMoveToTheRecycleBinToolbarItem_ to  _addDeleteToolbarItem_).

Reply:
I believe the meaning is the same, we are only changing the label. This is still the Move to the recycle bin action, isn't it?

Reply:
Ok, I see your point. Then we can leave the method name as it is... thanks!